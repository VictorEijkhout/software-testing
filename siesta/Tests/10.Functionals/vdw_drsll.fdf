%include ../base_graph.fdf
SystemLabel     vdw_drsll

xc.functional   VDW
xc.authors      DRSLL

ReparametrizePseudos T
