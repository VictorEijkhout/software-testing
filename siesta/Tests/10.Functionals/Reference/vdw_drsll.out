Siesta Version  : 5.0.0-beta1-27-gee499a791-dirty
Architecture    : ----
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch;-O3 -march=native
PP flags        : ----
Libraries       : ----
Parallelisations: none
NetCDF support
NetCDF-4 support
Lua support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/Tests/10.Functionals/work/vdw_drsll
* Running in serial mode.
>> Start of run:  13-DEC-2023  15:51:56

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

reinit: Reading from standard input
reinit: Dumping input in INPUT_TMP.77191
************************** Dump of input data file ****************************
%include ../../base_graph.fdf
SystemLabel     vdw_drsll
xc.functional   VDW
xc.authors      DRSLL
ReparametrizePseudos T
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: Graphite
reinit: -----------------------------------------------------------------------
reinit: System Label: vdw_drsll
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:    6 Label: C


---- Processing specs for species: C
Ground state valence configuration:   2s02  2p02
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/C.psf

---- Pseudopotential check for C 

Pseudized shells:
2s( 2.00) rc: 1.49
2p( 2.00) rc: 1.52
3d( 0.00) rc: 1.58
Valence configuration for ps generation: (assumed as above)
For C, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.
C pseudopotential only contains V_ls up to l=2 -- lmxkb reset.

<basis_specs>
===============================================================================
C                    Z=   6    Mass=  12.011        Charge= 0.17977+309
Lmxo=2 Lmxkb= 2    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=2
          i=1  nzeta=1  polorb=0  (2s)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=1  polorb=1  (2p)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=2  Nsemic=0  Cnfigmx=3
          i=1  nzeta=0  polorb=0  (3d)  (perturbative polarization orbital) (from 2p)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Filter: Requested cutoff:   50.000000 Ry
Using PAO.SplitTailNorm
atom: Called for C                     (Z =   6)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    4.00000

xc_check: Exchange-correlation functional:
xc_check: VDW Dion-Rydberg-Schroeder-Langreth-Lundqvist
xc_check: WARNING: Pseudopotential generated with LDA CA functional
V l=0 = -2*Zval/r beyond r=  1.4801
V l=1 = -2*Zval/r beyond r=  1.5087
V l=2 = -2*Zval/r beyond r=  1.5628
All V_l potentials equal beyond r=  1.5612
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.5628
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside     17.810 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     40.589 Ry
atom: Maximum radius for chlocal:    1.86354
atom: Maximum radius for r*vlocal+2*Zval:    1.70915

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.745634   el= -1.037288   Ekb=  5.157725   kbcos=  0.305340
GHOST: No ghost state for L =  1
   l= 1   rc=  1.743879   el= -0.434062   Ekb= -4.319496   kbcos= -0.378904
GHOST: No ghost state for L =  2
   l= 2   rc=  1.934160   el=  0.013252   Ekb= -1.016246   kbcos= -0.011215

KBgen: Total number of Kleinman-Bylander projectors:   9
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 2s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.003675 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    5.006988
                 energy =   -1.033046
                kinetic =    0.864501
    potential(screened) =   -1.897547
       potential(ionic) =   -5.455390
PAO Filter: Cutoff used:   24.50 Ry
PAO Filter: diagnostic kin energy tol:    0.003000 Ry
PAO Filter: Nominal cutoff (before filtering):   27.00 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 26  6

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 2p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.003675 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    6.166982
                 energy =   -0.429833
                kinetic =    2.511510
    potential(screened) =   -2.941343
       potential(ionic) =   -6.384176
PAO Filter: Cutoff used:   24.50 Ry
PAO Filter: diagnostic kin energy tol:    0.003000 Ry
PAO Filter: Nominal cutoff (before filtering):   41.24 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 30  8

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 2p
Split based on tail norm

   izeta = 1
                     rc =    6.166982
                 energy =    1.072979
                kinetic =    2.225526
    potential(screened) =   -1.152547
       potential(ionic) =   -3.934817
PAO Filter: diagnostic kin energy tol:    0.003000 Ry
PAO Filter: Nominal cutoff (before filtering):   44.82 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 30  7
atom: Total number of Sankey-type orbitals:  9

atm_pop: Valence configuration (for local Pseudopot. screening):
 2s( 2.00)                                                            
 2p( 2.00)                                                            
 3d( 0.00)                                                            
Vna: chval, zval:    4.00000   4.00000

Vna:  Cut-off radius for the neutral-atom potential:   6.166982
VNA Filter: diagnostic kinetic energy tol:    0.003000 Ry
VNA Filter: Nominal cutoff (before filtering):   22.00 Ry
VNA Filter: Cutoff used:    50.00 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 38 12

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1    6 C                       # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
C                     2                    # Species label, number of l-shells
 n=2   0   1                         # n, l, Nzeta 
   5.007   
   1.000   
 n=2   1   1 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   6.167   
   1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file C.ion.nc
coor:   Atomic-coordinates input format  =     Fractional

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   0.00000  1        1
siesta:      0.00000   0.00000   6.33814  1        2
siesta:     -0.00000   2.68503   0.00000  1        3
siesta:      2.32531   1.34252   6.33814  1        4

siesta: System type = bulk      

initatomlists: Number of atoms, orbitals, and projectors:      4    36    36

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = none
redata: Number of spin components                   = 1
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        1
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = NO
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =    50.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =     1000
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   T
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   299.9863 K
redata: Fix the spin of the system                  =   F
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 2
mix.SCF:    Linear mixing weight                 =     0.250000
mix.SCF:    Mixing weight                        =     0.250000
mix.SCF:    SVD condition                        = 0.1000E-07
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.2500
  weight.linear 0.2500
  history 2
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:    16.000000
Total ionic charge:    16.000000
 Kpoints in:           25 . Kpoints trimmed:           25

siesta: k-grid: Number of k-points = 25
siesta: k-points from cutoff
siesta: k-cutoff (effective) =     6.152 Ang
siesta: k-point supercell and displacements
siesta: k-grid:     0     5     0      0.000
siesta: k-grid:     5     0     0      0.000
siesta: k-grid:     0     0     2      0.500

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000

superc: Internal auxiliary supercell:     9 x     9 x     3  =     243
superc: Number of atoms, orbitals, and projectors:    972   8748   8748


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

superc: Internal auxiliary supercell:     9 x     9 x     3  =     243
superc: Number of atoms, orbitals, and projectors:    972   8748   8748

outcell: Unit cell vectors (Ang):
        2.461000    0.000000    0.000000
       -1.230500    2.131289    0.000000
        0.000000    0.000000    6.708000

outcell: Cell vector modules (Ang)   :    2.461000    2.461000    6.708000
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000    120.0000
outcell: Cell volume (Ang**3)        :     35.1841
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=36 nrows=36 sparsity=56.5833 nnzs=73332, refcount: 7>
  <dData1D:(new from dSpData1D) n=73332, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...

Attempting to read DM from file... Failed...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=36 nrows=36 sparsity=56.5833 nnzs=73332, refcount: 8>
  <dData2D:DM n=73332 m=1, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      82     381

InitMesh: MESH = 10 x 10 x 30 = 3000
InitMesh: Mesh cutoff (required, used) =    50.000    55.279 Ry
New grid distribution [1]: sub = 2

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -240.797496
siesta: Eions   =       997.163120
siesta: Ena     =       197.091253
siesta: Ekin    =       413.590332
siesta: Enl     =       -44.366348
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =         0.438164
siesta: DUscf   =         0.670030
siesta: DUext   =         0.000000
siesta: Ex      =      -168.788332
siesta: Ec      =       -21.804606
siesta: Exc     =      -190.592939
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -614.494888
siesta: Etot    =      -620.332627
siesta: FreeEng =      -620.332627

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1     -614.494888     -620.332627     -620.332627  1.362203 -5.766484  0.282568
timer: Routine,Calls,Time,% = IterSCF        1       0.123   4.85
   scf:    2     -620.339141     -620.335929     -620.335929  0.003282 -5.698138  0.193191
   scf:    3     -620.342105     -620.339222     -620.339222  0.006772 -5.548263  0.013453
   scf:    4     -620.339266     -620.339245     -620.339245  0.000485 -5.548430  0.010048
   scf:    5     -620.339290     -620.339269     -620.339269  0.001100 -5.548882  0.001934
   scf:    6     -620.339271     -620.339270     -620.339270  0.000203 -5.549005  0.000722
   scf:    7     -620.339271     -620.339270     -620.339270  0.000098 -5.549098  0.000239

SCF Convergence by DM+H criterion
max |DM_out - DM_in|         :     0.0000977008
max |H_out - H_in|      (eV) :     0.0002388856
SCF cycle converged after 7 iterations

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      82     381

siesta: E_KS(eV) =             -620.3393

siesta: E_KS - E_eggbox =      -620.3393

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot    0.000000    0.000000   -0.000000
----------------------------------------
   Max    0.018925
   Res    0.010607    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    0.018925    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):     -320.31     -323.46       59.04       -0.00       -0.00       -2.73
(Free)E + p*V (eV/cell)     -616.0590
Target enthalpy (eV/cell)     -620.3393

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -241.642933
siesta: Eions   =       997.163120
siesta: Ena     =       197.091253
siesta: Ekin    =       412.508689
siesta: Enl     =       -44.199976
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =         1.149542
siesta: DUscf   =         0.666493
siesta: DUext   =         0.000000
siesta: Ex      =      -168.587129
siesta: Ec      =       -21.805022
siesta: Exc     =      -190.392151
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -620.339271
siesta: Etot    =      -620.339270
siesta: FreeEng =      -620.339270

siesta: Final energy (eV):
siesta:  Band Struct. =    -241.642933
siesta:       Kinetic =     412.508689
siesta:       Hartree =     164.756082
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:         Exch. =    -168.587129
siesta:         Corr. =     -21.805022
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =    -190.392151
siesta:  Ion-electron =    -737.646443
siesta:       Ion-ion =    -269.565447
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =    -620.339270
siesta:         Fermi =      -5.549098

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.199921   -0.001702   -0.000000
siesta:    -0.001702   -0.201886   -0.000000
siesta:    -0.000000   -0.000000    0.036850

siesta: Cell volume =         35.184138 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00132496          0.00132702  Ry/Bohr**3
siesta:           0.12165252          0.12184171  eV/Ang**3
siesta:         194.90882706        195.21193362  kBar
(Free)E+ p_basis*V_orbitals  =        -618.496374
(Free)Eharris+ p_basis*V_orbitals  =        -618.496374

cite: Please see "vdw_drsll.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.0.0-beta1-27-gee499a791-dirty
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =       3.603
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       3.602       3.602   100.00
Setup                  1       2.027       2.027    56.29
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1       1.573       1.573    43.68
geom_init              1       0.381       0.381    10.57
state_init             1       0.068       0.068     1.88
hsparse                1       0.007       0.007     0.19
overlap                1       0.036       0.036     1.00
Setup_H0               1       0.189       0.189     5.25
naefs                  2       0.000       0.000     0.01
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.021       0.043     1.19
nlefsm                 2       0.118       0.236     6.55
DHSCF_Init             1       0.029       0.029     0.79
DHSCF1                 1       0.001       0.001     0.04
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.027       0.027     0.75
REMESH                 1       0.004       0.004     0.10
REORD                 26       0.000       0.000     0.00
PHION                  1       0.021       0.021     0.59
COMM_BSC              39       0.000       0.000     0.00
POISON                10       0.000       0.001     0.01
fft                   20       0.000       0.000     0.01
setup_H                8       0.063       0.507    14.07
DHSCF                  9       0.078       0.704    19.55
DHSCF3                 9       0.062       0.558    15.49
rhoofd                 9       0.024       0.213     5.90
XC                     9       0.020       0.180     4.99
GXC-CellXC             9       0.020       0.180     4.99
gridxc@cellXC          9       0.020       0.180     4.98
gridxc@fft3d         540       0.000       0.007     0.18
vmat                   9       0.018       0.165     4.57
IterSCF                7       0.123       0.863    23.95
compute_dm             7       0.067       0.472    13.11
diagon                 7       0.067       0.471    13.08
c-eigval             175       0.001       0.142     3.96
c-buildHS            175       0.001       0.125     3.47
cdiag                350       0.000       0.057     1.57
cdiag1               350       0.000       0.004     0.11
cdiag2               350       0.000       0.009     0.26
cdiag3               350       0.000       0.040     1.12
c-eigvec             175       0.001       0.164     4.56
cdiag4               175       0.000       0.001     0.03
c-buildD             175       0.001       0.163     4.54
MIXER                  6       0.000       0.003     0.07
PostSCF                1       0.330       0.330     9.15
DHSCF4                 1       0.146       0.146     4.07
dfscf                  1       0.144       0.144     3.99
overfsm                1       0.011       0.011     0.29
writeHSX               1       0.001       0.001     0.02
state_analysis         1       0.000       0.000     0.00
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.001       0.001     0.02
optical                1       0.000       0.000     0.00
  

>> End of run:  13-DEC-2023  15:52:00
Job completed
